lib/operations/command.rb

```diff
@@ -123,11 +122,11 @@
 class Operations::Command
   UNDEFINED = Object.new.freeze
   EMPTY_HASH = {}.freeze
-  COMPONENTS = %i[contract policies preconditions idempotency operation on_success].freeze
+  COMPONENTS = %i[contract policies preconditions idempotency operation on_success on_failure].freeze
```

Comment:
@radekzawada you didn't add `on_failure` component documentation, all components are described from line 70 to 117